School.js

```diff
@@ -8,60 +8,60 @@ class School {
       //   physics: {} 
     }
   }
-
-  /**
-   * Add class to classes
-   * 
-   * @param {string} name - Name of the class
-   * @param {string} teacher - Name of instructor 
-   * @return {Class} Class object
-   */
+  // /**
+  //  * Add class to classes
+  //  * 
+  //  * @param {string} name - Name of the class
+  //  * @param {string} teacher - Name of instructor 
+  //  * @return {Class} Class object
+  //  */
   addClass(name, teacher) {
     let newClass = new Class(name, teacher);
     this.classes[name] = newClass;
```

Comment:
You need to return `newClass` here so that you could use that return value to include it in your response